delfin/drivers/ibm/storwize_svc/ssh_handler.py

```diff
@@ -391,3 +396,186 @@ def fix_alert(self, alert):
+        for i in range(1, 3):
+            port_array = []
+            port_command = 'lsportip %s' % i
+            port_info = self.exec_ssh_command(port_command)
+            port_arr = port_info.split('\n')
+            port_map = {}
+            for detail in port_arr:
+                if detail is not None and detail != '':
+                    strinfo = detail.split(' ', 1)
+                    key = strinfo[0]
+                    value = ''
+                    if len(strinfo) > 1:
+                        value = strinfo[1]
+                    port_map[key] = value
+                else:
+                    if len(port_map) > 1:
+                        port_array.append(port_map)
+                        port_map = {}
+                        continue
+            for port in port_array:
+                if port.get('failover') == 'yes':
+                    continue
+                status = constants.PortHealthStatus.ABNORMAL
+                if port.get('state') == 'online':
+                    status = constants.PortHealthStatus.NORMAL
+                conn_status = constants.PortConnectionStatus.DISCONNECTED
+                if port.get('link_state') == 'active':
+                    conn_status = constants.PortConnectionStatus.CONNECTED
+                port_type = constants.PortType.ISCSI
+                location = '%s_%s' % (port.get('node_name'),
+                                      port.get('id'))
+                port_result = {
+                    'name': port.get('id'),
+                    'storage_id': storage_id,
+                    'native_port_id': port.get('id'),
+                    'location': location,
+                    'connection_status': conn_status,
+                    'health_status': status,
+                    'type': port_type,
+                    'max_speed': int(self.handle_port_bps(port.get('speed'))),
+                    'native_parent_id': port.get('node_name'),
+                    'mac_address': port.get('MAC'),
+                    'ipv4': port.get('IP_address'),
+                    'ipv4_mask': port.get('mask'),
+                    'ipv6': port.get('IP_address_6'),
+                    'ipv6_mask': port.get('mask')
+                }
+                port_list.append(port_result)
+
+    def handle_port_bps(self, value):
+        speed = 0
+        if value:
+            if value.isdigit():
+                speed = float(value)
+            else:
+                unit = value[-4:-2]
+                speed = float(value[:-4]) * int(
+                    self.change_capacity_to_bytes(unit))
+        return speed
+
+    def list_ports(self, storage_id):
+        try:
+            port_list = []
+            self.get_fc_port(port_list, storage_id)
+            self.get_iscsi_port(port_list, storage_id)
+            return port_list
+        except Exception as err:
+            err_msg = "Failed to get ports metrics from Storwize: %s" % \
```

Comment:
Please use 'attributes ' or 'propperties' instead of 'metrics ' in messages..If we mention metrics it will lead to misunderstanding to user it is a performance metric collection

Reply:
Done